sw/device/lib/testing/test_framework/check.h

```diff
@@ -100,13 +100,13 @@
         if (!fail) {                                                          \
           /* Print a failure message as soon as possible. */                  \
           if (OT_VA_ARGS_COUNT(_, ##__VA_ARGS__) == 0) {                      \
-            LOG_ERROR("CHECK-fail: " #actual_ " does not match " #expected_); \
+            LOG_INFO("CHECK-fail: " #actual_ " does not match " #expected_); \
```

Comment:
Are the updates in this file intended?

Reply:
Yes, I'm not sure if thats the best solution, but using `LOG_ERROR` was aborting the test with dv_sim and not priting the array.

Reply:
I think you use the default max_quit_count, it's recommended to use a bigger number for debug, like this `dvsim.py ... +max_quit_count=50`. The default setting is to save compute resource so that the simulation is terminated at the 1st error.

I'm fine to change some of them to LOG_INFO, perhaps keep the one in 109 where it already prints the mismatch values.

Reply:
I didn't know about `+max_quit_count=50`, so I'm reverting the changes and keeping it as it is, thanks.